ZAPD/Main.cpp

```diff
@@ -33,13 +33,11 @@ void BuildAssetTexture(const std::string& pngFilePath, TextureType texType,
                        const std::string& outPath);
 void BuildAssetBackground(const std::string& imageFilePath, const std::string& outPath);
 void BuildAssetBlob(const std::string& blobFilePath, const std::string& outPath);
-void BuildAssetModelIntermediette(const std::string& mdlPath, const std::string& outPath);
+void BuildAssetModelIntermediette(const std::string& outPath);
 void BuildAssetAnimationIntermediette(const std::string& animPath, const std::string& outPath);
 
-int NewMain(int argc, char* argv[]);
-
 #if !defined(_MSC_VER) && !defined(__CYGWIN__)
-void ErrorHandler(int sig)
+void ErrorHandler(int seg)
```

Comment:
I may be wrong, but I think `sig` comes from `signal`.

Reply:
That should be fixed in the latest commit but ill check when I do format.

Reply:
You're right. My bad :sweat_smile: